index.html

```diff
@@ -38,39 +46,48 @@ <h2>I am a Software Developer</h2>
   </section>
 
   <section class="works-section">
-    <h2>My Recent Works</h2>
-    <hr>
+    <div id="works-header">
+      <h2>My Recent Works</h2>
+      <hr>
+    </div>
     <div class="cards-container">
       <div class="card">
```

Comment:
- [ ] Please, consider fixing the previous review suggestion regarding the hovered project. To implement it without JS, using CSS could be a solution. 👍

Reply:
Hi @elmar8287 

Thank you for pointing that out. I have implemented it using CSS. Can you kindly check if it meets the requirement?

Reply:
> Hi @elmar8287
> 
> Thank you for pointing that out. I have implemented it using CSS. Can you kindly check if it meets the requirement?
> 
> ![image](https://user-images.githubusercontent.com/20554583/161382906-6c9336ea-4ade-478d-acb1-09cf75f6d924.png)

Hi @emmiiorji currently this feature is not implemented properly, just as the first reviewer requested

Reply:
Hi @prolajumokeoni.

Thank you for your response.

Please, can you specify the feature not properly implemented? However, if it has to do with linking the demo or GitHub repo of the project, the projects currently showing are only dummies.

Reply:
The hover effect for the cards, please check my comment below 🤗